oaknational/Oak-Web-Application
# refactor: card variants and adds card as link component

## Description

- Adds CardAsLink component
- Refactors CardImageIcon variant
- Tidy up of storybook Card section


## How to test

1. Go to {[cloud link](https://samara-storybook-d1xs8l71c-oak-national-academy.vercel.app/?path=/story/introduction--page)}
2. Click on Card subheading
3. You should see Card, CardAsLink and CardVariants with examples in each from [figma](https://www.figma.com/file/96yUIKNRvg0rMCWasdkJ1u/Design-System-2.0?node-id=1939%3A16548) designs


## Checklist
